packages/datadog_flutter_plugin/test/datadog_sdk_test.dart

```diff
@@ -247,7 +247,11 @@ void main() {
 
     expect(datadogSdk.firstPartyHosts.length, 2);
     expect(datadogSdk.firstPartyHosts[0].hostName, 'example.com');
+    expect(datadogSdk.firstPartyHosts[0].headerTypes,
+        {TracingHeaderType.datadog, TracingHeaderType.tracecontext});
```

Comment:
can you also assert the data in both headers (datadog and tracecontext) is same?

if this is covered somewhere else, I'm fine with that also.

Reply:
👍 I actually thought I had it in the actual tracing library, but it appears not. I'll add it.

Reply:
Added for the main library. Ready for re-review.